torch/_subclasses/fake_tensor.py

```diff
@@ -978,7 +983,13 @@ def __torch_dispatch__(cls, func, types, args=(), kwargs=None):
         # because the next dispatch after a fake mode will attempt to use
         # subclasses of tensors to dispatch, and any FakeTensor arguments
         # will be considered eligible.
-        if any(not issubclass(t, FakeTensor) and t is not torch.Tensor for t in types):
+        unrecognized_types = [
+            t for t in types if not issubclass(t, FakeTensor) and t is not torch.Tensor
+        ]
+        if unrecognized_types:
+            not_implemented_log.debug(
+                "FakeTensor unrecognized subclass(es): %s", unrecognized_types
+            )
```

Comment:
(doesn't matter for this PR) should we audit all `NotImplemented` returns and add this logging in more places 🤔

Reply:
I audited all the ones on fake and proxy, which are the ones I care the most about.